src/something_entity.cpp

```diff
@@ -134,7 +119,9 @@ void Entity::render(SDL_Renderer *renderer, Camera camera, SDL_Color shade) cons
     } break;
 
     case Entity_State::Poof: {
-        poof.render(renderer, camera.to_screen(pos), camera.to_screen(texbox_world()), flip);
+        Rectf texbox = poof_animat.transform_rect(texbox_local, pos);
+        // TODO: Poof state loses last alive frame
```

Comment:
I don't really understand the problem here. Could you please elaborate (preferably in [the body of the TODO](https://github.com/tsoding/snitch/tree/4a50e49501a94a7528c36b6f9d48f4b86504a534#todo-body) so it becomes the part of the reported issue).